src/xcode/ENA/ENA/Source/HTTPClient/Resources/SIDValidationDecoratorResource.swift

```diff
@@ -4,7 +4,7 @@
 
 import Foundation
 
-struct ServiceIdentityDocumentValidationDecoratorResource: Resource {
+struct SIDValidationDecoratorResource: Resource {
```

Comment:
I would prefer not to introduce abbreviations, I think the name was more precise and easier to understand before

Reply:
yes i just reverted it now